config/pai.service

```diff
@@ -0,0 +1,9 @@
+#sudo nano /etc/systemd/system/pai.service
+[Unit]
+Description=ParadoxAilarmInterface
+
+[Service]
+ExecStart=/home/ha/pai/run.py
```

Comment:
It is helpful to add some comments so that users change this setting.
Also, maybe we should refer to /opt/pai (outside the home of a specific user)